charts/prometheus-mysql-exporter/README.md

```diff
@@ -23,10 +23,30 @@ $ helm install [RELEASE_NAME] prometheus-community/prometheus-mysql-exporter
 $ helm install --name [RELEASE_NAME] prometheus-community/prometheus-mysql-exporter
 ```
 
-_See [configuration](#configuration) below._
+_See [configuration](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-mysql-exporter#configuration) below._
 
 _See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
 
+## Upgrading Chart
+
+```console
+# Helm 3 or 2
+$ helm upgrade [RELEASE_NAME] [CHART] --install
+```
+
+### To =< 1.0.0
+
+Version 1.0.0 is a major update.
+
+* The chart now follows the new Kubernetes label recommendations:
+<https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/>
+
+The simplest way to update is to do a force upgrade, which recreates the resources by doing a delete and an install.
```

Comment:
Another option seems to be to remove selectors https://github.com/jenkinsci/helm-charts/issues/46.

Haven't tested it.

Reply:
Ups, sorry. Was'nt aware of the order. 
I'll put it back.

Reply:
Thanks this reads much clearer now